onnx2torch/utils/safe_shape_inference.py

```diff
@@ -12,7 +12,11 @@ def _is_big_model(model: ModelProto) -> bool:
     return model.ByteSize() / (1024 * 1024 * 1024) > 2.0
 
 
-def _shape_inference_by_model_path(model_path: Union[Path, str], output_path: [Path, str], **kwargs) -> ModelProto:
+def _shape_inference_by_model_path(  # pylint: disable=missing-function-docstring
```

Comment:
pylint annotation is not necessary and should be removed, because function is private '_'.

Reply:
Done, thanks @ivkalgin !